src/client/utils/view-definition-printer/view-definition-printer.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017-2018 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ViewDefinition, ViewDefinitionVersion } from "../../../common/view-definitions";
+
+export class ViewDefinitionPrinter {
+  private dataCubeName: string;
+  private viewDefinitionVersion: ViewDefinitionVersion;
+  private viewDefinition: ViewDefinition;
+
+  constructor(dataCubeName: string, viewDefinitionVersion: ViewDefinitionVersion, viewDefinition: ViewDefinition) {
```

Comment:
You can use typescript shortcut for defining constructor:

```
constructor(private dataCubeName: string, private viewDefinitionVersion: ViewDefinitionVersion, private viewDefinition: ViewDefinition) {}
```

That way you don't need to define fields on class and assign arguments for corresponding fields.